packages/helm/symphony/files/azure/prepare-namespaces.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/env bash
```

Comment:
Where do we use prepare-namepsaces.sh? If this file is Azure only, please keep in extension repo and patch them in build.